docs/product-user-guide/src/main/asciidoc/drl-rules-central-create-proc.adoc

```diff
@@ -0,0 +1,182 @@
+KieSession kieSession = kiebase.newKieSession();
+kieSession.setGlobal( "myGlobalList", list );
+----
++
+*<@Michael (5 of 13): Per question above, where does the user add the above snippet for the decision engine?>*
++
+* `function`: (optional) Use this to include a function to be used by rules in the DRL file. Functions put semantic code in your rule source file and enable you to invoke actions on the action (`then`) part of a rule. Functions are especially useful if an action (`then`) part of a rule is used repeatedly and only the parameters differ for each rule. Above the rules in the DRL file, you can declare the function or import a static method as a function, and then use the function by name in an action (`then`) part of the rule.
++
+.Declaring and using a function with a rule (option 1)
+[source,java]
+----
+function String hello(String applicantName) {
+    return "Hello "+applicantName+"!";
+}
+
+rule "Using a function"
+  when
+    eval( true )
+  then
+    System.out.println( hello( "James" ) );
+end
+----
++
+.Importing and using the function with a rule (option 2)
+[source,java]
+----
+import function my.package.applicant.hello;
+
+rule "Using a function"
+  when
+    eval( true )
+  then
+    System.out.println( hello( "James" ) );
+end
+----
++
+* `query`: (optional) Use this to search the decision engine for facts related to the rules in the DRL file. Queries search for a set of defined conditions and do not require `when` or `then` specifications. Query names are global to the KIE base and therefore must be unique among all other rule queries in the project. To return the results of a query, construct a traditional `QueryResults` definition using `ksession.getQueryResults("name")`, where `"name"` is the query name. This returns a list of query results, which allow you to retrieve the objects that matched the query. Define the query and query results parameters above the rules in the DRL file.
++
+.Query and query results for people under the age of 21, with a rule
+[source,java]
+----
+query "people over the age of 30"
+    person : Person( age < 21 )
+end
+
+QueryResults results = ksession.getQueryResults( "people under the age of 21" );
+System.out.println( "we have " + results.size() + " people under the age  of 21" );
+
+System.out.println( "These people are are under 21:" );
+}
+
+rule "Underage"
+  when
+    application : LoanApplication( )
+    Applicant( age < 21 )
+  then
+    application.setApproved( false );
+    application.setExplanation( "Underage" );
+end
+----
++
+* `declare`: (optional) Use this to declare a new fact type to be used by rules in the DRL file. The default fact type in DRL files is `Object`, but you can declare other types as needed. Declaring fact types in DRL files enables you to define a new fact model directly in the decision engine, without creating models in a lower-level language like Java. *<@Michael (6 of 13): This right? From my deductions with all the info out there.>*
```

Comment:
@sterobin Close enough ;-) The default fact type available for authoring are defined in ```java.lang``` (see [8.5.1 Import](https://docs.jboss.org/drools/release/7.5.0.Final/drools-docs/html_single/index.html#_import)). ```Object``` is in that package (but so are other types).